tests/wpunit/Tribe/JSON_LD/AbstractTest.php

```diff
@@ -55,16 +55,28 @@ public function it_should_return_an_empty_array_if_trying_to_get_data_for_empty(
 		$this->assertEquals( array(), $this->make_instance()->get_data( $empty ) );
 	}
 
+	/**
+	 * @test
+	 * it should be instantiatable
+	 */
+	public function user_has_cap_filter( $allcaps = [], $caps = [] ) {
```

Comment:
This looks like a left-over from debugging.
If out need to have a user that can do everything, in the test method use:
```php
wp_set_current_user(static::factory()->user->create(['role' => 'administrator']));
```

Reply:
Thanks, updated now.